client/state/ui/guided-tours/contexts.js

```diff
@@ -34,6 +34,15 @@ export const isEnabled = feature => () =>
 	config.isEnabled( feature );
 
 /**
+ * Returns true if there is a global `isDebuggingGuidedTours` present with a
+ * truthy value.
+ *
+ * @return {Boolean} True if `window.isDebuggingGuidedTours` truthy
+ */
+export const isDebuggingGuidedTours = () =>
```

Comment:
I played around with it a bit and then realized that a situation that I had especially with the editor tour a few times was that I was able to trigger broken behavior by reloading. But that resets `window`, so maybe a cookie (or a disjunction of both) would be more useful? WDYT?